### PR TITLE
Using central_config_dir to locate seafdav.conf

### PR DESCRIPTION
**Pull request by foxel**

### CODE REVIEW

**Comment by freeplant** [comment]
What's the reason for those modification?

**Comment by lins05** [comment]
I guess it's related to https://github.com/haiwen/seafile/issues/1543 .

File: controller/seafile-controller.c [hunk 1]
```diff
@@ -320,7 +320,7 @@ setup_env ()
     g_setenv ("SEAFILE_CENTRAL_CONF_DIR", ctl->central_config_dir, TRUE);
 
     char *seahub_dir = g_build_filename (installpath, "seahub", NULL);
-    char *seafdav_conf = g_build_filename (topdir, "conf", "seafdav.conf", NULL);
+    char *seafdav_conf = g_build_filename (ctl->central_config_dir, "seafdav.conf", NULL);
```
Comment (lins05) [inline controller/seafile-controller.c]: The change doesn't make any functional difference, but it LGTM to use the `central_config_dir` variable here.

**Comment by foxel** [comment]
Change is aimed on accounting config directory location while reading DAV server config as it's done for seafile service. 

There issue pops up when you place actual binaries in place other than actual top dir holding the data and config.

**Pull request merged**